SU-HSDO/stanford_mrc
# Frontend fixes

# READY FOR REVIEW

# Summary
- Featured Event Card now leveraged flex shrinking and growing as needed to ensure no overlapping of content
- Mobile search is now clickable
- Mobile menu now has active states

# Needed By (Date)
- End of sprint (3/29)

# Urgency
- N/A

# Steps to Test
1. Pull this branch
2. Verify that the above three items is working (can test against screenshots in Slack in related convo we had)

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- N/A

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
